### DISCUSSION TITLE
術語列表中 function 的翻譯可否改為「函數」

### DISCUSSION THREAD

**Discussion by husohome** [python/python-docs-zh-tw#533] [General]
術語詞表中 function 被翻譯為「函式」
https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8
但是我記得台灣的專有名詞應該是「函數」
有鑑於程式中的function與高中數學的函數概念其實應該是一樣的
是否翻成函數比較好？
按：教育部樂詞網
https://terms.naer.edu.tw/detail/8ce941ac47b81ecd356beb763332a8d9/?seq=4

**Comment by mattwang44** [comment]
我個人覺得應該維持「函式」，原因和[這篇](https://groups.google.com/g/chinese-l10n/c/wTqwhbaaS4g)和[這篇](https://hackmd.io/@sysprog/c-function)的簡介部分相仿：

- 多數著名繁中著作都翻譯為「函式」
- 應和數學的「函數」有所區別
